src/ray/protobuf/usage.proto

```diff
@@ -130,4 +130,12 @@ enum TagKey {
   // Name of Tune scheduler algorithm or "Custom" if user-defined.
   // Example: "FIFOScheduler"
   TUNE_SCHEDULER = 502;
+  // (AIR) Fully user-controlled experiment tracking integrations ("0" or "1")
```

Comment:
When will it ever be 0?

Reply:
Oh nope, I used the pattern from the SERVE tags above:

```
  // Whether or not a DAGDriver was used ("1" if used).
  SERVE_DAG_DRIVER_USED = 10;
```